src/Elasticsearch/EndpointAndIndexesConfigurator.php

```diff
@@ -120,6 +73,12 @@ private function handleAssetIndices(ConfigReader $configReader): void
             $this->indexManager->getIndexName(IndexManager::INDEX_ASSET, $endpointName),
             $this->assetMapping->generate($configReader->toArray())
         );
+
+        // Multi Assets
```

Comment:
We can't add multi asset. This is extension in our code. We need to introduce some interface

Reply:
That only creates empty index this shouldn't be a problem

Reply:
This `  public const INDEX_MULTI_ASSET = 'multiasset';` for sure is problem. It can't be in this bundle.